src/Swarm/TUI/Controller.hs

```diff
@@ -109,8 +109,8 @@ handleEvent = \case
   AppEvent (UpstreamVersion ev) -> do
     let logReleaseEvent l e = runtimeState . eventLog %= logEvent l ("Release", -7) (T.pack $ show e)
     case ev of
-      Left e@(FailedReleaseQuery _e) -> logReleaseEvent ErrorTrace e
-      Left e -> logReleaseEvent Said e
```

Comment:
@kostmo @byorgey FYI: this used to create nice yellow warning logs by abusing the fact that _messages said by the robot with ID `-7` happen to be in yellow._ :sweat_smile: